.travis.yml

```diff
@@ -22,34 +21,40 @@ env:
 branches:
   only:
     - master
+    - 2.0-beta
     # npm version tags
     - /^v\d+\.\d+\.\d+/
 
 jobs:
-  fail_fast: true
+  fast_finish: true
   allow_failures:
     - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-lts-2.8
```

Comment:
Why are we using both Travis and GitHub Actions? Shouldn't we use just one or the other? My preference would be GitHub Actions.

Reply:
Oh yeah this file should just get deleted. I thought I had already done it, but I've been bouncing around so many repos maybe I forgot.

Reply:
Indeed. We use GH Actions as a "regular" CI and Travis to catch the possible issues with "floating" dependencies. I agree we should standardise on one; I would not consider it blocking 2.0. Open to discussing of course! 😄

Reply:
I think let's delete Travis for now and go with GH actions. I've got a floating dependencies job setup here if we want to copy that https://github.com/shipshapecode/ember-shepherd/blob/master/.github/workflows/main.yml#L25

Reply:
https://github.com/html-next/vertical-collection/pull/329